api-catalog-ui/frontend/src/selectors/selectors.jsx

```diff
@@ -59,6 +59,11 @@ export const getFilteredServices = (tiles, searchCriteria) => {
 
             tile.services = filteredServices.sort((service1, service2) => service1.title.localeCompare(service2.title));
             return compareResult(searchCriteria, tile, filteredServices);
-        })
-        .sort((tile1, tile2) => tile1.title.localeCompare(tile2.title));
+        });
 };
+
+export function sortServices(services) {
```

Comment:
function accepts services but then it works with tiles, it is also used with `tiles` in Dashboard